src/include/expression/operator_expression.h

```diff
@@ -48,6 +48,22 @@ class OperatorExpression : public AbstractExpression {
         return (
             type::ValueFactory::GetBooleanValue(type::PELOTON_BOOLEAN_NULL));
     }
+    if (exp_type_ == ExpressionType::OPERATOR_IS_NULL) {
+      PL_ASSERT(children_.size() == 1);
+      type::Value vl = children_[0]->Evaluate(tuple1, tuple2, context);
+      if (vl.IsNull())
+        return (type::ValueFactory::GetBooleanValue(true));
+      else
+        return (type::ValueFactory::GetBooleanValue(false));
+    }
+    if (exp_type_ == ExpressionType::OPERATOR_IS_NOT_NULL) {
```

Comment:
Actually, this code is redundant. Put `OPERATOR_IS_NULL` and `OPERATOR_IS_NOT_NULL` together in the same if branch then examine the `exp_type_` to check whether the return value of `IsNull()` should be true or false.